sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 4.1.0-preview.2 (Unreleased)
 
+- Using `Poller` and `PollerLike` from the latest `core-lro` instead of relying on the temporary `KVPoller` and `KVPollerLike`.
```

Comment:
`KVPoller` and `KVPollerLike` are not temporary anymore now that we are adding the type alias...

Reply:
Good point!